src/addresses/index.ts

```diff
@@ -7,24 +7,30 @@ import { connectorsV1 as connectorsV1_Mainnet } from './mainnet/connectorsV1'
 import { connectorsV2_M1 as connectorsV2_M1_Mainnet } from './mainnet/connectorsV2_M1'
 import { connectorsV1 as connectorsV1_Polygon } from './polygon/connectorsV1'
 import { connectorsV2_M1 as connectorsV2_M1_Polygon } from './polygon/connectorsV2_M1'
+import { connectorsV1 as connectorsV1_Arbitrum } from "./arbitrum/connectorsV1"
 
 export const Addresses = {
-    genesis: '0x0000000000000000000000000000000000000000',
-    core,
-    connectors: {
-        chains: {
-            1: {
-                versions: {
-                    1: connectorsV1_Mainnet,
-                    2: connectorsV2_M1_Mainnet 
-                }
-            },
-            137: {
-                versions: {
-                    1: connectorsV1_Polygon,
-                    2: connectorsV2_M1_Polygon
-                }
-            }
+  genesis: '0x0000000000000000000000000000000000000000',
+  core,
+  connectors: {
+    chains: {
+      1: {
+        versions: {
+          1: connectorsV1_Mainnet,
+          2: connectorsV2_M1_Mainnet,
         },
-    }
+      },
+      137: {
+        versions: {
+          1: connectorsV1_Polygon,
+          2: connectorsV2_M1_Polygon,
+        },
+      },
+      42161: {
+        versions: {
+          1: connectorsV1_Arbitrum
```

Comment:
```suggestion
          2: connectorsV2_Arbitrum
```

@ishanjain28 We don't have v1 connectors for arbitrum. Only v2 connectors